index.js

```diff
@@ -1,88 +1,129 @@
 // ***** Question 1 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
 
-// console.log(counter) 
+let counter = 1
+
+counter = 2
+
+
+console.log("%cQuestion 1", "color: red") 
+console.log(counter) 
 // // => 2
-// console.log("%c----------", "color: red") 
+
+console.log("%c----------", "color: red") 
 
 
 // ***** Question 2 *****
 // *** Uncomment the lines below to test
 // *** after testing, comment the line causing the error back in 
 // *** otherwise, the error will stop the rest of your code from running
-// console.log("%cQuestion 2", "color: red") 
 
-// console.log(name)
+const name = "Johnson" 
+console.log("%cQuestion 2", "color: red") 
+
+console.log(name)
 // => "Raffy"
 // name = "Not Raffy" 
 // => TypeError
-// console.log("%c----------", "color: red") 
+console.log("%c----------", "color: red") 
 
 
 
 // ***** Question 3 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 3", "color: red") 
+console.log("%cQuestion 3", "color: red") 
 
-// console.log(drinkWater(12))
+function drinkWater(currentThirstLevel) {
+  console.log("Man I sure am thirsty")
+  console.log("Ahh that hits the spot")
+  console.log(currentThirstLevel -= 1) 
+}
+
+drinkWater(12)
```

Comment:
This is correct - but if you want to `return` the value of `currentThirstLevel`, you can subtract one from the current thirst like you had, and then `return currentThirstLevel`!